schema/va-spec/base/json/InSilicoVariantEffectPredictionProposition

```diff
@@ -0,0 +1,122 @@
+{
+   "$schema": "https://json-schema.org/draft/2020-12/schema",
+   "$id": "https://w3id.org/ga4gh/schema/va-spec/1.0.1/base/json/InSilicoVariantEffectPredictionProposition",
+   "title": "InSilicoVariantEffectPredictionProposition",
+   "maturity": "draft",
+   "type": "object",
+   "description": "A Proposition describing the predicted impact of a variant based on an variant effect predictor.",
```

Comment:
This sounds a bit circular. How about - 
'A Proposition describing the impact of a variant as predicted by an in-silico tool'
to match ACMG/AMP language?